suite-common/wallet-core/src/transactions/transactionsThunks.ts

```diff
@@ -87,6 +101,144 @@ export const replaceTransactionThunk = createThunk(
+            blockHeight: 0,
+            blockTime: Math.floor(new Date().getTime() / 1000),
+            confirmations: 0,
+            fees: '0',
+            hex: '0',
+            valueIn: '0',
+            blockHash: '',
+        });
+
+        // todo: this part (updating sending account), could well be part of forEach lower, but there is a fixture
+        // in useSendForm.test ("Success with: custom fee, 2 outputs, 0 utxo (ignored)")
+        // that won't pass in that case and I did not want to change that
+        const pendingAccount = getPendingAccount(account, transaction, txid);
+        if (pendingAccount) {
+            dispatch(accountsActions.updateAccount(pendingAccount));
+            const accountTransaction = blockbookUtils.transformTransaction(
+                account.descriptor,
+                account.addresses,
+                blockbookTransaction,
+            );
+            const prependingTx = { ...accountTransaction, deadline: blockHeight + 2 };
+            dispatch(
+                transactionsActions.addTransaction({
+                    transactions: [prependingTx],
+                    account,
+                }),
+            );
+        }
+
+        // only 1 fake tx may be created per affected account,
+        const affectedAccounts = blockbookTransaction.vout.reduce<{
+            [affectedAccountKey: string]: Account;
+        }>((result, output) => {
+            if (output.addresses) {
+                // todo: reduce addresses
+                findAccountsByAddress(output.addresses[0], accounts).forEach(affectedAccount => {
+                    // sending account is always affected and is solved above
+                    if (affectedAccount.key === account.key) return accounts;
+                    if (!result[affectedAccount.key]) {
+                        result[affectedAccount.key] = affectedAccount;
+                    }
+                });
+            }
+            // output.address_n -> change -> ignore
+            return result;
+        }, {});
+
+        Object.keys(affectedAccounts).forEach(key => {
+            const affectedAccount = affectedAccounts[key];
+            const accountTransaction = blockbookUtils.transformTransaction(
+                affectedAccount.descriptor,
+                affectedAccount.addresses,
+                blockbookTransaction,
+            );
+            const prependingTx = { ...accountTransaction, deadline: blockHeight + 2 };
+            dispatch(
+                transactionsActions.addTransaction({
+                    transactions: [prependingTx],
+                    account: affectedAccount,
+                }),
+            );
+            if (affectedAccount.backendType === 'coinjoin') {
+                // updating of coinjoin accounts is solved in coinjoinAccoundActions and coinjoinMiddleware
+                return;
+            }
+            const pendingReceivingAccount = getPendingReceivingAccount(
```

Comment:
sendFormActions? should action stop after actual adding tx?

Reply:
using only updated version of getPendingAccount now